frontend/src/Loader.tsx

```diff
@@ -1,15 +1,25 @@
 import React, { ReactElement } from "react";
+import { Die } from "./domain/Die";
+
+export interface LoaderProps {
+  dieType: Die;
+}
+
+export const Loader = ({ dieType }: LoaderProps): ReactElement => {
+  // Type aliases cannot be used as a parameter type, so no [key: Die]
+  // in other words....no dice! £pun
+  const shapeMap: { [key: string]: string } = {
```

Comment:
could use `Record<Die, string>` instead to keep the typing

Reply:
Changed!

Reply:
An issue with this approach is that: 
<img width="331" alt="Screen Shot 2020-12-06 at 4 20 58 PM" src="https://user-images.githubusercontent.com/6589964/101292731-5a784c00-37df-11eb-99f4-3c4d5d65a943.png"> doesn't throw any errors.... so enforcing the first thing to be a `Die` doesn't actually do what we want it do it 😞